### PR TITLE
Swagger 3 upgrade

### PR DESCRIPTION
**Pull request by jakehschwartz**

### CODE REVIEW

**Comment by av8data** [comment]
Won't this change to the release.yml cause any branch to be released?

**Comment by jakehschwartz** [comment]
@av8data I thought that syntax it would only happen on a tag push

**Comment by mgd43b** [comment]
It all seems to compile just fine, but i'm getting some NPEs that i'm trying to track down when running:

I also seem to be getting the same when running "hello-world-example/test"

Caused by: java.lang.NullPointerException
	at com.jakehschwartz.finatra.swagger.FinatraOperation.pathParam(FinatraOperation.scala:23)

Caused by: java.lang.NullPointerException
	at com.jakehschwartz.finatra.swagger.FinatraOperation.respondsWith(FinatraOperation.scala:110)

File: src/main/scala/com/twitter/finatra/http/SwaggerRouteDSL.scala [hunk 1]
```diff
@@ -2,17 +2,18 @@ package com.twitter.finatra.http
...
   * To work around the accessibility of RouteDSL, this class is in "com.twitter.finatra.http" package
   */
 object SwaggerRouteDSL {
-  implicit def convert(dsl: RouteDSL)(implicit swagger: Swagger): SwaggerRouteDSL = new SwaggerRouteDSLWrapper(dsl)(swagger)
+  implicit def convert(dsl: RouteDSL)(implicit swagger: FinatraSwagger): SwaggerRouteDSL = new SwaggerRouteDSLWrapper(dsl)(swagger)
 }
 
 trait SwaggerRouteDSL extends RouteDSL {
-  implicit protected val swagger: Swagger
+  implicit protected val swagger: FinatraSwagger
```
Comment (mgd43b) [inline src/main/scala/com/twitter/finatra/http/SwaggerRouteDSL.scala]: consider renaming for consistency:
implicit protected val swagger: FinatraSwagger

to 

implicit protected val finatraSwagger: FinatraSwagger

**Comment by jakehschwartz** [comment]
@mgd43b Got rid of all the NPEs but now theres nothing in the json file that gets generated hahaha. Will continue trying

**Comment by jakehschwartz** [comment]
@mgd43b Getting closer, the only problem I am seeing is with the AddressModel ref

**Comment by jakehschwartz** [comment]
@mgd43b I think I got it working. If you could try it locally to make sure I didnt break anything, that would be a big help. 

Otherwise I just may merge it and deal with the bugs later :)

**Comment by mgd43b** [comment]
@jakehschwartz  - this is working for me locally, thanks!

**Pull request merged**